play.pokemonshowdown.com/src/panel-popups.tsx

```diff
@@ -1512,6 +1708,30 @@ class RoomTabListPanel extends PSRoomPanel {
 		</div></PSPanelWrapper>;
 	}
 }
+class BattleTimerPanel extends PSRoomPanel {
+	static readonly id = 'battletimer';
+	static readonly routes = ['battletimer'];
+	static readonly location = 'semimodal-popup';
+	static readonly noURL = true;
+
+	triggerTimer = (ev: Event) => {
+		const room = this.props.room.getParent() as BattleRoom;
+		let value = (ev.currentTarget as HTMLButtonElement).value || '';
+		if (value === "starttimer") room.send("/timer start");
+		if (value === "stoptimer") room.send("/timer stop");
+		this.close();
+	};
+
+	override render() {
+		const room = this.props.room.getParent() as BattleRoom;
+		return <PSPanelWrapper room={this.props.room}><div class="pad">
+			{ room.timerInterval ?
+				<button class="button" value="stoptimer" onClick={this.triggerTimer}> Stop Timer</button> :
+				<button class="button" value="starttimer" onClick={this.triggerTimer}> Start Timer</button>}
```

Comment:
```suggestion
			{room.timerInterval ? (
				<button class="button" data-cmd="/closeand /timer stop"> Stop Timer</button>
			) : (
				<button class="button" data-cmd="/closeand /timer start"> Start Timer</button>
			)}
```

Oh I guess the problem is that the `/timer start` goes in the wrong room. Maybe we need a `/closeandinopener`.

Reply:
Yeah, but do we really need it? I don't think there are enough use cases to justify it, and to be honest now that i think about it, the logic for what happens after clicking the button should remain within the popup panel method imo

Reply:
oh nvm, i ignored the fact that this button only needs to send 1 command in a specifric room